### PR TITLE
Colorize

### PR DESCRIPTION
**Pull request by ehmry**
Just a bit of color coding of types.

### CODE REVIEW

File: nix-repl.cc [hunk 1]
```diff
@@ -645,19 +654,19 @@ std::ostream & NixRepl::printValue(std::ostream & str, Value & v, unsigned int m
...
         break;
 
     case tPrimOpApp:
-        str << "«primop-app»";
+        str << ESC_BLU "«primop-app»" ESC_END;
         break;
 
     default:
-        str << "«unknown»";
+        str << "ESC_RED «unknown»" ESC_END;
```
Comment (solson) [inline nix-repl.cc]: The first `"` is in the wrong place here.

**Comment by ehmry** [comment]
@solson fixed at 34ec981

**Comment by solson** [comment]
Looks good to me. @edolstra Want to merge?

**Comment by domenkozar** [comment]
Could you attach a screenshot how it looks like? That should increase chances to merge this PR :)

**Comment by ehmry** [comment]
Of course,
[asciicast](https://asciinema.org/a/79250)

**Pull request merged**